.github/workflows/SHiELD_parallelworks.yml

```diff
@@ -0,0 +1,45 @@
+name: Compile SHiELD SOLO and run tests
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  buildstep:
+    runs-on: [self-hosted]
+    name: SOLO SHiELD build
+    strategy:
+      fail-fast: true
+      max-parallel: 1
```

Comment:
How many runners do you have?  You have 4 jobs, do they need to all run one after the other?  You could set up 4 runners to run all 4 at the same time.

Reply:
I only have 1 right now.  But I will set up multiple runners so that we can do multiple tests at the same time.  For the build step I need them all to build in series (and not parallel).

Reply:
I believe the builds could be done in parallel if we have the CI system build the FMS library first.

Reply:
Would there be an issue with the builds occurring simultaneously when they all use the same Build/exec folder? I could alter the Build system so that they all build with different exec folders

Reply:
The current build system uses exec/${config}_${compiler}/  It could be altered to use a combination of _config, hydro, comp, bit, and compiler_

Reply:
I will start making those changes - building in parallel would make everything much faster.

Reply:
Remember to run the FMS build as a standalone process.  Once we bring in the updates to build the few NCEPlibs needed by SHiELD, that would become part of that first step.